zetaclient/bitcoin_client.go

```diff
@@ -459,14 +459,14 @@ func (ob *BitcoinChainClient) PostGasPrice() error {
 	if feeResult.Errors != nil || feeResult.FeeRate == nil {
 		return fmt.Errorf("error getting gas price: %s", feeResult.Errors)
 	}
-	gasPrice := big.NewFloat(0)
-	gasPriceU64, _ := gasPrice.Mul(big.NewFloat(*feeResult.FeeRate), big.NewFloat(1e8)).Uint64()
+	feeRate := new(big.Int).SetInt64(int64(*feeResult.FeeRate * 1e8))
+	feeRatePerByte := new(big.Int).Div(feeRate, big.NewInt(1000))
```

Comment:
@lumtis I'm all for that. If something can be fixed in zetacore/smart contract, we shouldn't bother it in zetaclient because Mock main can iterates in a faster pace.

Reply:
I'll creat a constant `bytesPerKB` for unification.